modules/spatiotemporal/test/test_make_graph.py

```diff
@@ -0,0 +1,170 @@
+                self.assertAlmostEqual(trajectory[1], 2.0, delta=1e-4)
+
+    def test_composition_scoring(self):
+        # tests composition scoring functionality. Found in composition_scoring.py, calc_likelihood
+        state_dict, expected_subcomplexes, exp_comp_map = setup_system()
+        # set input dir
+        with IMP.test.temporary_directory() as tmpdir:
+            input = os.path.join(tmpdir, 'data')
+        shutil.copytree(self.get_input_file_name('data'), input)
+        graph, graph_scores = spatiotemporal.create_DAG(state_dict, input_dir=input, out_cdf=False, dag_heatmap=False,out_labeled_pdf=False, score_comp=True,exp_comp_map=exp_comp_map,draw_dag=False,spatio_temporal_rule=False,expected_subcomplexes=expected_subcomplexes)
+        # Check the overall score for the lowest energy trajectory
+        for trajectory in graph_scores:
+            if trajectory[0][0].get_label() == '1' and trajectory[0][1].get_label() == '2' and trajectory[0][2].get_label() == '1':
+                self.assertAlmostEqual(trajectory[1], 6.701131199228036, delta=1e-4)
+
+    def test_writing_output(self):
+        # tests writing text output. From write_output.py
+        state_dict, expected_subcomplexes, exp_comp_map = setup_system()
+        # set input dir
+        with IMP.test.temporary_directory() as tmpdir:
+            input = os.path.join(tmpdir, 'data')
+        shutil.copytree(self.get_input_file_name('data'), input)
+        # set output dir
+        output=self.get_tmp_file_name('output')
+        spatiotemporal.create_DAG(state_dict, input_dir=input,output_dir=output, dag_heatmap=False,out_labeled_pdf=True,out_cdf=True,out_pdf=True,draw_dag=False)
+        # Read in input files
+        cdf=np.loadtxt(output+'/cdf.txt')
+        # the 2nd most likely path in the cdf
+        self.assertAlmostEqual(cdf[1], 0.42117519, delta=1e-4)
+        # the 2nd most likely path in the pdf
+        pdf = np.loadtxt(output + '/pdf.txt')
+        self.assertAlmostEqual(pdf[1], 0.2105876, delta=1e-4)
+        check_label_pdf=open(output+'/labeled_pdf.txt','r')
+        # read in the 2nd line,
+        line=check_label_pdf.readline()
+        line=check_label_pdf.readline()
+        line_split=line.split()
+        check_label_pdf.close()
+        self.assertEqual(line_split[0], '1_0min|2_5min|2_10min|')
+
+    def test_writing_dag(self):
+        # tests writing DAG output. From write_output.py
+        state_dict, expected_subcomplexes, exp_comp_map = setup_system()
+        # set input dir
+        with IMP.test.temporary_directory() as tmpdir:
+            input = os.path.join(tmpdir, 'data')
+        shutil.copytree(self.get_input_file_name('data'), input)
+        # set output dir
+        output=self.get_tmp_file_name('output')
+        skip=0
+        try:
+            from graphviz import Digraph
+        except ImportError:
+            print('graphviz not available, will not test drawing graph')
+            skip=1
```

Comment:
Use the `skipTest` method to skip a test. It will show up as skipped in the test outputs. See https://docs.python.org/3/library/unittest.html#unittest.TestCase.skipTest